{"name": "ASCII assert message in step", "status": "passed", "steps": [{"name": "Given feature definition ru", "status": "passed", "attachments": [{"name": ".text", "source": "97a498ea-d82d-4415-abd5-73d906a9b922-attachment.txt", "type": "text/plain"}], "start": 1568945804420, "stop": 1568945804429}, {"name": "When I run behave with allure formatter with options \"--lang ru\"", "status": "passed", "start": 1568945804429, "stop": 1568945804467}, {"name": "Then allure report has a scenario with name \"Ошибка с utf-8 сообщением\"", "status": "passed", "start": 1568945804468, "stop": 1568945804468}, {"name": "And scenario contains step \"Допустим провальный шаг с ascii\"", "status": "passed", "start": 1568945804469, "stop": 1568945804469}, {"name": "And step has status details with message \"AssertionError: Фиаско!\"", "status": "passed", "start": 1568945804470, "stop": 1568945804470}], "start": 1568945804418, "stop": 1568945804470, "uuid": "5bf3b72d-4c6b-4a54-9ef0-ff2ce156a490", "historyId": "1d10a61acd28a38689d6544b702fb1e4", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Language"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}